common/djangoapps/third_party_auth/tests/test_identityserver3.py

```diff
@@ -0,0 +1,50 @@
+"""
+Unit tests for the IdentityServer3 OAuth2 Backend
+"""
+import unittest
+from third_party_auth.identityserver3 import IdentityServer3
+from third_party_auth.tests import testutil
+
+
+@unittest.skipUnless(testutil.AUTH_FEATURE_ENABLED, testutil.AUTH_FEATURES_KEY + ' not enabled')
```

Comment:
Please use either single or double quotes for consistency
```suggestion
@unittest.skipUnless(testutil.AUTH_FEATURE_ENABLED, testutil.AUTH_FEATURES_KEY + " not enabled")
```